README.rst

```diff
@@ -100,6 +100,28 @@ field, containing a dictionary of options. For example:
 The client takes the key/value pairs and writes out the ``RecaptchaOptions``
 value in JavaScript.
 
+Within a Form Wizard
+~~~~~~~~~~~~~~~~~~~~
+
+If you're intending to use the reCaptcha field in a form wizard you can optionally pass the
+``wizard`` parameter to the constructor. For example:
+
+.. code-block:: python
+
+    captcha = ReCaptchaField(
+        wizard=True,
+    )
+
+This will cache the successful captcha validation response and prevent subsequent validation
+failures when the wizard reruns validation on your forms.
+
+The result will be cached for 10 minutes by default.
+If you require a custom expiration period, add a ``RECAPTCHA_WIZARD_EXPIRY`` setting, for example:
+
+.. code-block:: python
+
+   RECAPTCHA_WIZARD_EXPIRY = 30
```

Comment:
Document that it's minutes, not seconds. The Django convention is to specify everything in seconds by the way.